be/CMakeLists.txt

```diff
@@ -1060,6 +1060,11 @@ install(FILES
     ${BASE_DIR}/../conf/udf_security.policy
     DESTINATION ${OUTPUT_DIR}/conf)
 
+install(FILES
+        ${BASE_DIR}/../conf/hdfs-site.xml
```

Comment:
Right now in `basedir../conf/` there is no `hdfs-site.xml` and `core-site.xml` file.

Reply:
The 'OPTIONAL' keyword ensures that errors will not occur without these two files.